pyalgomate/brokers/__init__.py

```diff
@@ -318,3 +269,17 @@ def getBroker(feed, api, broker, mode, capital=200000):
             brokerInstance = LiveBroker(api)
 
     return brokerInstance
+
+
+def getDefaultUnderlyings() -> list[str]:
+    return  ['NSE:NIFTY BANK']
```

Comment:
This file is common for all brokers, Can you move this method to zerodha file if its specific to zerodha?

Reply:
I wanted to generalize the code. I have handled the change for finvasia to replace from : to |, similar logic can be applied to Kotak if needed.